### PR TITLE
Linkedin profile

### PR DESCRIPTION
**Pull request by Vaibhavpratapsingh22**
Chapter-1 👍

### CODE REVIEW

File: 11-linkedin/1-profile.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+# Chapter - 1 Profile
+LinkedIn is a social network specifically designed for career and business professionals to connect. Over 65 million professionals use LinkedIn to cultivate their careers and businesses. Unlike other social networks in which you might become "friends" with anyone and everyone, LinkedIn is about building strategic relationships. Thus, the number of connections is less important than the type of connections. In fact, LinkedIn stops showing your actual number of connections once you have 500 because it's about quality, not quantity. 
+
+
+##  Picture And Cover
```
Comment (udeysingh95) [inline 11-linkedin/1-profile.md:5]: this topic suddenly comes from nowhere - correct and connect the flow of content in this chapter.

File: 11-linkedin/1-profile.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Your profile picture is very important because on the basis of your profile picture people will want to connect to you or offer you business proposals or maybe offer you a job. Now, if you don't have a profile picture or if it's outdated than you really have to work on that. What do you need to have a good profile picture? Well, first thing is that is has to be close up. Look into the camera, smile, have a neutral background and dress appropriately, wearing minimal jewellery and accessories and certainly no head-wear or no sunglasses.\
+These are things to avoid in your profile
+picture like shades, bad lightning etc.\
+So how do you know that your profile picture is good? You could ask colleagues or maybe friends or family members and make changes according to their response and reaction on the picture.
+
+<p text align="center"></p><img src="https://www.thebalancecareers.com/thmb/EptDIw2uysv-KHddKl0poWA6q7Q=/1500x0/filters:no_upscale():max_bytes(150000):strip_icc():format(webp)/take-a-professional-photo-for-linkedin-2064035-v4-5babf1fac9e77c0025592c55.png"></p>
```
Comment (udeysingh95) [inline 11-linkedin/1-profile.md:11]: this image says the same things - as written above in the text. 
Better way of using image would be to show - what are examples of bad pictures and good pictures for linkedin - both cover and profile.

File: 11-linkedin/1-profile.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+##  Headline
```
Comment (udeysingh95) [inline 11-linkedin/1-profile.md:13]: This topic - is quite important and it plays an important role in SEO of your profile. 
you didn't mention that at all. Also - some examples are recommended to add in each section - as the theory part is known to everyone - but how it looks like in the end - is a big question mark to all.

File: 11-linkedin/1-profile.md [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+Your headline is the piece of text that comes below your name and you have to see that as an ad as the tile of an ad. It is the thing which will attract people to read open your profile and read about you.
+So there's a simple rule that you can use to enhance to make your title better on Linkedin.The rule is **Job title + Company name + most important keywords + Zing factor**. Now let me explain that, of course you have your job title and then your company name but it's also very important to ad the important keywords. Now why is that? Because when people go on Linkedin they often try to find certain profiles. They use the search option to find certain profiles and they search for a keyword like sales executive.So if you have this keyword in your headline the serch will show you to the people. Now comes the Zing factor, that's something that surprises people, that could be a conversation starter, that explains a little bit more what you stand for. 
+
+## Summary
```
Comment (udeysingh95) [inline 11-linkedin/1-profile.md:17]: update the content - there is no more linking available in the summary section. its only text. Also, there are no tips to write summary anywhere - everyone knows already - what summary is! and that is what you have written again. 
maybe show some good and bad summaries also - if you can design a template - use that.

Comment (udeysingh95) [reply 11-linkedin/1-profile.md:17]: We dont want to learn here - what and where to write summary in linkedin profile - we want to learn - how to write an eye catching summary on your linkedin profile. Change the whole content in this section as well.

**Review by udeysingh95 — CHANGES_REQUESTED** [review]
This is a special chapter - feel free to write it as long as you need and want. I see a big deficiency of content on this PR - this PR is only talking about what is section "x" and what is section "y"! This isn't the purpose of this PR. 
The purpose is to learn how to create and put nice, attractive, professional content in those sections and what are the good practises.

**Comment by udeysingh95** [comment]
## Read more.

**Comment by udeysingh95** [comment]
also - topic-0 "url" is not mentioned

File: 11-linkedin/README.md [hunk 5]
```diff
@@ -0,0 +1,11 @@
...
+All-Star rating, that means your profile is optimized for success. So what are you waiting for! start now.
+## Overview
+LinkedIn, business-oriented social networking Web site founded in 2002 and headquartered in Mountain View, California. In 2016 LinkedIn was acquired by Microsoft for approximately $26 billion. The following year it reported more than 500 million members in about 200 countries.LinkedIn, business-oriented social networking Web site founded in 2002 and headquartered in Mountain View, California. Unlike other social networks such as Facebook, instagram which are often purely recreational, LinkedIn emphasizes a user’s professional connections. Users create profile pages that have a structure similar to a résumé, in that users can summarize their career, advertise their particular skills, and list their education and employment history. Connections are formed between users when one accepts an invitation from another to join his or her network. LinkedIn allows users to further their careers by searching for jobs, finding connections (even at third hand) at a particular company, and receiving recommendations from other users.
```
Comment (SaarthakDeshmukh) [inline 11-linkedin/README.md:5]:
```suggestion
LinkedIn, business-oriented social networking Web site founded in 2002 and headquartered in Mountain View, California. In 2016 LinkedIn was acquired by Microsoft for approximately $26 billion. The following year it reported more than 500 million members in about 200 countries. LinkedIn, business-oriented social networking Web site founded in 2002 and headquartered in Mountain View, California. Unlike other social networks such as Facebook, instagram which are often purely recreational, LinkedIn emphasizes a user’s professional connections. Users create profile pages that have a structure similar to a résumé, in that users can summarize their career, advertise their particular skills, and list their education and employment history. Connections are formed between users when one accepts an invitation from another to join his or her network. LinkedIn allows users to further their careers by searching for jobs, finding connections (even at third hand) at a particular company, and receiving recommendations from other users. 
```

File: 11-linkedin/1-profile.md [hunk 6]
```diff
@@ -0,0 +1,92 @@
...
+
+I suggest either keeping your headline chocked full of keywords, with a bit of finessing copy (“I’m a digital marketing strategist that help small businesses reach more customers online.”) or taking the time to craft a compelling headline with conversion copywriting.
+
+<p text align="center"><img src="https://searchengineland.com/figz/wp-content/seloads/2019/04/image14.png"></p>
+
+In the example above, this business owner is straight to the point by simply listing what her job title is and the services she offers. This is fine. It includes proper keywords that could potentially draw in people that are looking for services like hers.
+
+If you wish to continue the reading [click here](https://searchengineland.com/seo-guide-to-optimizing-your-linkedin-profile-for-more-connections-better-leads-315882#:~:text=Your%20headline%20is%20the%20line,this%20guide%20and%20compelling%20copy.).
+
+## Summary
```
Comment (SaarthakDeshmukh) [inline 11-linkedin/1-profile.md:36]: The image is looking nice. I can take some inspiration to write my LinkedIn summary.

File: 11-linkedin/1-profile.md [hunk 7]
```diff
@@ -0,0 +1,92 @@
...
+
+## Profile URL
+Before heading towards the profile picture, let us first fic the URL. When you first sign up for LinkedIn, it will assign you a generic URL this is your name, dashes and numbers. It’s impossible to remember or share easily. Luckily, you can change it with just a few clicks and keystrokes.
+
+Use your first and last name, or you can use a URL that you have used elsewhere, such as your Facebook personal profile. Just choose something that uses your name and is easy to remember. That’s the main point of the vanity URL—it gives people something easy to remember. 
+
+LinkedIn lets you change it at any time, but pick something good and stick with it. Whenever you change your URL, you’ll lose people who have you bookmarked.
+
+## Adding Profile and Background Picture
```
Comment (SaarthakDeshmukh) [inline 11-linkedin/1-profile.md:11]: This section is nicely explained.

**Review by SaarthakDeshmukh — CHANGES_REQUESTED** [review]
Overall I could understand everything. This will help a lot of people as you are gathering all the relevant information in one place. Great Work @Vaibhavpratapsingh22

**Review by udeysingh95 — APPROVED** [review]

**Pull request merged**